primitives/sandbox/Cargo.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-wasmi = { version = "0.9.0", default-features = false, features = ["core"] }
+wasmi = { version = "0.9.1", default-features = false, features = ["core"] }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 wasmi = "0.9.0"
```

Comment:
Any idea why the dependabot only cares about the wasm32 dependencies?

Reply:
Maybe a bug? :D

Reply:
Yeah the root problem is that wasmi does not comply with cargo best practices. Created an issue for this: https://github.com/paritytech/wasmi/issues/270 . For now we should just merge this.